tasks/friends-app.md

```diff
@@ -0,0 +1,61 @@
+[![MIT Licensed][icon-mit]][license]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Telegram][icon-chat]][chat]
+
+# Friends App
+
+When you're done with your challenge push your code to github repo and publish
+it using `gh-pages`. _Add link to it to your main frontend tasks repo._
+
+If you honestly finished all the previous steps than go ahead and share it with
+others - send a message in [course channel](chat) with the link to you repo and the words "Weather APP - finished" and add link to repo with code of the
+website.
```

Comment:
I think we also need version of the site on gitpages. Then we can twit them, like it was with weather app. It is good for motivation and for our channel in Twitter

Reply:
totally agree, change the name of the app.